packages/umi-test/src/index.js

```diff
@@ -59,6 +59,8 @@ export default function(originOpts = {}) {
       ...(userModuleNameMapper || {}),
     },
     testPathIgnorePatterns: ['/node_modules/'],
+    // 用于设置 jest worker 启动的个数
+    maxWorkers: process.env.MAX_WORKERS ? Number(process.env.MAX_WORKERS) : '',
```

Comment:
不清楚 macWorkers 设为 `''` 有啥影响，这样保险点。

```suggestion
    ...(process.env.MAX_WORKERS ? { maxWorkers: Number(process.env.MAX_WORKERS) } : {}),
```

Reply:
已修改